HW_12/src/order.ts

```diff
@@ -0,0 +1,28 @@
+import { Meal } from "./meel";
+
+export class Order {
+    customerName: string;
+    orderItems: Meal[];
+    private totalPrice: number;
+    orderId?: number;

```

Comment:
What for orderId here if you have id in IBarOrder?

Reply:
it was a missclick, i used fast fixing by IDE. Fixed the issue